tools/wasm-sourcemap.py

```diff
@@ -233,12 +240,13 @@ def main():
     wasm = append_source_mapping(wasm, options.source_map_url)
 
   if options.w:
-    print('Saving wasm to %s' % options.w)
+    logging.debug('Saving wasm to %s' % options.w)
     with open(options.w, 'wb') as outfile:
       outfile.write(wasm)
 
-  print('Done.')
+  return 0
```

Comment:
Why not both? Print "Done." as well as return 0

Reply:
No reason.  Done.